schemas/variant_calling.json

```diff
@@ -424,7 +426,8 @@
         "type": "string",
         "enum": [
           "Germline",
-          "Somatic"
+          "Somatic",
+          "Germline+Somatic+Unknown"
```

Comment:
Did we decide to use `Somatic/Germline`?

Reply:
I'm indifferent, @guanqiaofeng  did you have a preference?

Reply:
Both look good to me. I am slightly inclined towards `Somatic/Germline` as it matches better with the file names, which contain `somatic-germline`